includes/admin/auth-service-actions.php

```diff
@@ -40,10 +36,16 @@ public function __construct()
 	{
 		// Set an option if the user rated the plugin.
 		add_action('wp_ajax_oauth_deauthenticate_site', [$this, 'oauth_deauthenticate_site']);
-		//add_action('wp_ajax_nopriv_oauth_deauthenticate_site', [$this, 'oauth_deauthenticate_site']);
-		// Do toke check here.
-		$post_type = get_post_type();
-		if($post_type == 'calendar'){
+
+		$post_type = '';
+
+		if (isset($_GET['post_type']) && 'calendar' === $_GET['post_type']) {
+			$post_type = esc_attr($_GET['post_type']);
+		} elseif (isset($_GET['post']) && !empty($_GET['post'])) {
+			$post_id = tesc_attr($_GET['post']);
+			$post_type = get_post_type($post_id);
+		}
+		if (isset($post_type) && !empty($post_type) && 'calendar' === $post_type) {
```

Comment:
```suggestion
		if ('calendar' === $post_type) {
```
This should be enough, right?

Reply:
Update code